src/blocks/content-helper/content-helper.scss

```diff
@@ -1,313 +1,312 @@
-@import "variables";
 @import "../shared/variables";
 @import "../shared/functions";
 
+.wp-parsely-content-helper {
 
-.parsely-spinner-wrapper {
-	display: flex;
-	justify-content: center;
-	margin-top: to_rem(40px);
+	.parsely-spinner-wrapper {
```

Comment:
Most of the changes in this file are indentation differences (done to avoid too many indentation levels). I have moved a couple of things around and done some limited re-organizing.